srml/balances/src/lib.rs

```diff
@@ -88,6 +86,17 @@
 //!
 //! - `vesting_balance` - Get the amount that is currently being vested and cannot be transferred out of this account.
 //!
+//! ### Signed Extensions
+//!
+//! The balances module defines the following extensions:
+//!
+//!   - [`TakeFees`]: Consumes fees proportional to the length and weight of the transaction.
+//!     Additionally, it can contain a single encoded payload as a `tip`. The inclusion priority
+//!     is increased proportional to the tip.
+//!
+//! Lookup the runtime aggregator file (e.g. `node/runtime`) to see the full list of signed
```

Comment:
I don't see any reason to write this sentence?

Reply:
I have a strong incentive to mention it as much as I can: so that the user would not think that just defining this struct and implementing the trait is enough and we, like it is magic, take care of the rest. In reality, doing ^^ does exactly _nothing_. Once included in the runtime's `UncheckedExtrinsic` type then it starts doing stuff :D

Similar to modules that get aggregated somewhere, extensions also get combined at some point in the runtime file and I find it an important note to point out. Hence, let's keep such sentences.

Reply:
Why not write this? You write someone should look into the runtime to see all signed extensions. You should write that the user needs to add them to the extrinsics type to be used.